jak-na-konflikty.md

```diff
@@ -0,0 +1,110 @@
+
+ * Sedni si sám v tiché místnosti na 5 minut a zamedituj si o tvém kolegovi. Soustřeď se na jeho kladné vlastnosti a hezké věci, co jste spolu zažili. Uvědom si, že nacházející konflikt je pravděpodobně něco, čemu se za půl roku nad pivem zasmějete.
+ * Jdi na chvíli pracovat nebo dělat něco jiného. Dělej něco, co ti dělá radost a obnoví tvoji víru v dobro lidí.
+
+
+## Vyvolání řešení konfliktu
+
+Zastav kolegu v soukromí a popros ho, jestli by si na tebe na chvilku neudělal čas. Řekni mu, že s ním potřebuješ vyřešit nějaké pnutí (klidně to můžeš říct přátelštější či vtipnější formou). Pokud čas nemá, domluv se na čase, kdy se to bude oběma stranám hodit.
+
+Pokud kolega nechce řešit konflikt s tebou o samotě, vyhledejte rovnou mediátora.
+
+## Konflikt ve skupině
+
+Může se stát, že má s kolegou konflikt celá skupina. Tato situace je velmi složitá ke korektnímu řešení a zejména velmi nekomfortní pro konfrontovaného kolegu. Měl by si se proto držet těchto pravidel:
+
+* Pokud máš s kolegou osobní konflikt, nezatahuj do něj další kolegy
+* Neřešte kolegovi věci za zády
+* Ber ohled na konfrontovaného, pokud budete řešit konflikt pod nátlakem, může se stát, že bude řešení neefektivní.
+
+Vyvolání komnfliktu za skupinu by měl dělat jeden zástupce a měl by kolegovi zvědomit, že se jedná a skupinový konflikt.
+
+V případě skupinového konfliktu je potřeba začínat řešení přímo s mediátorem.
+
+## Řešení konfliktu
+
+Konflikt řeš někde, kde budete mít soukromí a když na to budete mít oba dostatečné množství času.
+
+Postupuj volně dle následujícíh bodů. Popros kolegu aby ti naslouchal a nepřerušoval tě.
+
+1. **Tohle se stalo / Tohle se děje** => Popiš bez emocí předmět konfliktu. 
+2. **Takhle se kolem toho cítím** => Popiš jaký má na tebe předmět konfliktu emocionální dopad. Pokus se svoje emoce regulovat tak, aby to ten druhý ustál.
+3. **Proč si myslím, že to bylo/je špatně** =>  Zvědom, proč podle tebe bylo jednání tvého kolegy špatně. (pokud to není absolutně zřejmé)
+4. **Potřeboval bych, aby se tohle stalo / změnilo** => Popiš, jakou očekáváš u kolegy změnu v jeho chování/jeho přístupu.
+
+Teď je řada na tobě abys kladl otázky ty, nepřerušoval a naslouchal tvému kolegovi. Upozaď ego a snaž se kolegu pochopit.
+
+1. **Co se podle tebe stalo? / Co se děje?** => Zeptej se kolegy jak předmět konfliktu vidí on. 
+2. **Jak se kolem toho cítíš?** => Zeptej se kolegy, jaké má kolem celé situace emoce.
+3. **Pokud se mnou neshouhlasíš, proč si myslíš, že to bylo/je dobře?** => Zeptej se kolegy, proč si myslí, že postupoval dobře? Snaž se do něj vcítit a jeho motivaci pochopit.
+4. **Co ode mě potřebuješ, aby se to stalo / změnilo** => Zeptej se kolegy, co potřebuje od tebe, aby ti byl schopný vyhovět.
+
+Společně se domluvte na dalším postupu.
+
+Vyjádři svůj respekt a kolegovi poděkuj za ochotu a odvahu konflikt otevřeně a upřímně řešit.
+
+Nezapomínej, že konflikt je soukromá věc. Tato schůzka je důvěrná, pokud se s kolegou nedohodnete jinak.
```

Comment:
@Vratislav @Vratislav @prokopsimek Možná by stálo za to sem ještě přidat doporučení, aby pokud dojdou ke konsenzu se domluvili na nějaké době, kdy si udělají retrospektivu?

Reply:
Dobrej nápad. Přidal jsem